.github/workflows/utils.yml

```diff
@@ -192,15 +193,15 @@ jobs:
         run: |
           mkdir build
           cd build
-          cmake ${{github.workspace}} -DBUILD_PLUGIN_NOOP=ON -DENABLE_TESTS=ON -DENABLE_COVERAGE=ON -DCMAKE_CXX_COMPILER=g++-12 -DCMAKE_C_COMPILER=gcc-12 -DGCOV_EXECUTABLE=gcov-12
+          cmake ${{github.workspace}} -DENABLE_TESTS=ON -DENABLE_COVERAGE=ON -DCMAKE_CXX_COMPILER=g++-12 -DCMAKE_C_COMPILER=gcc-12 -DGCOV_EXECUTABLE=gcov-12
           make
-          make test ARGS="-V"
+          CTEST_OUTPUT_ON_FAILURE=1 make test ARGS="-V"
           cd ../
         shell: bash
 
       - name: Calculate coverage
         run: |
-          gcovr --cobertura=./docs/coverage/coverage_pr.xml --gcov-executable gcov-12 -s -e "build/_deps/*"
+          gcovr --filter src/ --filter plugins/ --cobertura=./docs/coverage/coverage_pr.xml --gcov-executable gcov-12 -s -e "build/_deps/*"
```

Comment:
I think we can get rid of the  ```-e "build/_deps/*"``` here if we include the ```--filter```

Reply:
yep, makes sense